examples/js/loaders/LWOLoader.js

```diff
@@ -328,13 +328,13 @@ THREE.LWOLoader = ( function () {
 			params = Object.assign( maps, params );
 			params = Object.assign( params, attributes );
 
-			var type = connections.attributes.Roughness ? 'Standard' : 'Phong';
+			var materialCtor = connections.attributes.Roughness ? THREE.MeshStandardMaterial : THREE.MeshPhongMaterial;
 
-			return new THREE[ 'Mesh' + type + 'Material' ]( params );
```

Comment:
I had to change this piece since it's not module conform (you have no `THREE` namespace).